libcore/src/pedestrian/AgentsSourcesManager.cpp

```diff
@@ -53,12 +53,7 @@ void AgentsSourcesManager::Run()
     LOG_INFO("Starting agent manager thread");
     //Generate all agents required for the complete simulation
     //It might be more efficient to generate at each frequency step
-    //TODO  this loop is exactly GenerateAgents( --> REFACTOR)
-    for(const auto & src : _sources) {
-        LOG_INFO("Generate src: {}", src->GetId());
-        src->GenerateAgentsAndAddToPool(src->GetMaxAgents(), _building);
-    }
-
+    GenerateAgents();
```

Comment:
Are all pedestrian computed beforehand or only the ones added in this step? Is this comment still valid?

Reply:
This is just refactoring. Functionality was not altered. 
Here, for every source a `MaxAgents` are generated and added to Pool, waiting for `processAllSources` to be added to `building`.